ci-operator/step-registry/baremetalds/e2e/upgrade/test/baremetalds-e2e-upgrade-test-ref.yaml

```
@@ -0,0 +1,34 @@
+ref:
+  as: baremetalds-e2e-upgrade-test
+  from: tests
+  commands: baremetalds-e2e-upgrade-test-commands.sh
+  env:
+  - name: TEST_COMMAND
+    default: run
+    documentation: The test command to run.  Use 'openshift-test --help' to list available commands.
+  - name: TEST_SUITE
+    default: openshift/conformance/parallel
+    documentation: The test suite to run.  Use 'openshift-test TEST_COMMAND --help' to list available suites.
+  - name: TEST_SKIPS
+    default: ""
+    documentation: Regular expression (POSIX basic regular expression) of tests to skip.
+  - name: TEST_OPTIONS
+    default: ""
+    documentation: The test command options. Use 'openshift-test TEST_COMMAND --help' to list available options.
+  - name: TEST_INSTALL_CSI_DRIVERS
+    default: ""
+    documentation: The CSI driver(s) to use.  Used by the `openshift-tests` program, see its documentation for details.
+  - name: TEST_CSI_DRIVER_MANIFEST
+    default: ""
+    documentation: |-
+      Name of the CSI driver manifest file to use.  Used by the `openshift-tests`
+      program as TEST_CSI_DRIVER_FILES env. var., see its documentation for
+      details. The file must be present in ${SHARED_DIR}.
+  resources:
```

```suggestion
  dependencies:
  - name: "release:latest"
    env: OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE
  resources:
```
See https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-ref.yaml#L27-L29 - this sets env var with the release used to upgrade. In the periodic job [external script](https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/release/openshift-release-release-4.6-periodics.yaml#L5461-L5494) would create `release:latest` and `release:initial`